src/core/CCRandom.mli

```diff
@@ -145,3 +153,11 @@ val (<*>) : ('a -> 'b) t -> 'a t -> 'b t
 
 val run : ?st:state -> 'a t -> 'a
 (** Using a random state (possibly the one in argument) run a generator *)
+
+(** {6 Random generator testing } *)
```

Comment:
I would prefer, here, a `(**/**)` section, because this test is not really a part of the library. Additionally, a proper unit test (using qtest) would be nice, but I can add it myself later.